templates/haproxy_route.cfg.j2

```diff
@@ -0,0 +1,67 @@
+{% extends 'haproxy.cfg.j2' %}
+{% block proxy_configuration %}
+frontend haproxy
+    mode http
+    bind [::]:80 v4v6
+    bind [::]:443 v4v6 ssl crt {{ haproxy_crt_dir }}
```

Comment:
Maybe this is out of context of this PR but just a heads-up: with IRC Bridge using ingress, I saw this bind 443 being enabled before having the certificates, resulting in an error.

Reply:
That's strange, haproxy should block until certificates are available. Let's move this discussion internally

Reply:
I'll mark this as resolved until we find exactly how to reproduce.